source/initHQ/fortifyFOB.sqf

```diff
@@ -1,12 +1,17 @@
-_centerPos = _this select 3;
+_centerPos = (_this select 3) select 0;
 _actionID = _this select 2;
 _object = _this select 0;
+_fob = (_this select 3) select 1;
+aliveAllUnits = {alive _x} count allunits;
 
 if (commandpointsblu1 < 4) exitWith {hint "You don't have enough Command Points"};
 commandpointsblu1 = commandpointsblu1 - 4;
 publicvariable "commandpointsblu1";
-_object removeAction _actionID;
 
+
+if ((Warcom_Limiter_Param == 1) && (aliveAllUnits>99)) exitWith {hint "*DUWS AI Limiter is ENABLED!*\n\nTry Fortifying again when there are less than 100 AI units on the map"};
```

Comment:
I don't understand what you mean by making it check against configurable. That line just checks to see if there are more than 100 living units, and if there are, it won't allow fortification. It doesn't have any impact on performance as it only counts and it only takes a hundredth of a second to do. If you mean we should make it check dead bodies as well, fortifying would be impossible until the cleanup script goes through again and by then, OPFOR just spawned more units at their remaining zones.